test/unit/app-lifecycle.spec.js

```diff
@@ -99,6 +99,38 @@ describe('App-Lifecycle', function() {
     });
   });
 
+  describe('when restarting the app', function() {
+    beforeEach(function() {
+      this.myApp.start({
+        state: { 'foo': 'bar' }
+      });
+    });
+
+    it('should be stopped', function() {
+      this.myApp.restart();
+
+      expect(this.stopStub).to.have.been.calledOnce;
+    });
+
+    it('should trigger start event twice', function() {
```

Comment:
I think this it is misleading.  I don't we should say "when restarting the app it should trigger start event twice"  perhaps 'should re-trigger the start event' more appropriately describes what's happening in the feature.. or to go along with the previous `it` maybe 'should be started again`